Packages/com.unity.inputsystem/InputSystem/Editor/ControlPicker/InputControlPickerDropdown.cs

```diff
@@ -511,12 +511,12 @@ internal override string DrawSearchFieldControl(string searchString)
                     // When picking controls, have a "Listen" button that allows listening for input.
                     if (m_Owner.m_Mode == InputControlPicker.Mode.PickControl)
                     {
-                        using (new EditorGUILayout.VerticalScope(GUILayout.MaxWidth(42)))
+                        using (new EditorGUILayout.VerticalScope(GUILayout.MaxWidth(50)))
                         {
                             GUILayout.Space(4);
                             var isListeningOld = m_Owner.isListening;
                             var isListeningNew = GUILayout.Toggle(isListeningOld, "Listen",
-                                EditorStyles.miniButton, GUILayout.MaxWidth(45));
+                                EditorStyles.miniButton, GUILayout.MaxWidth(50));
```

Comment:
This one fixes a display issue with the new UI style overhaul on 2019.3 where the "Listen" button would be cut off in the middle of the last "n".